api/v1alpha1/networkfence_types.go

```diff
@@ -20,6 +20,16 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+type FenceState string
+
+const (
+	// PrimaryState represents the Primary replication state
```

Comment:
I guess the comments need to be changed, it does not match with the variables

Reply:
ACK